data/ecosystems/a/aptos.toml

```diff
@@ -121,8 +122,8 @@ url = "https://github.com/aptos-labs/aptos-ansible"
 url = "https://github.com/aptos-labs/aptos-canvas"
 
 [[repo]]
-url = "https://github.com/aptos-labs/aptos-core"
 tags = ["Protocol"]
+url = "https://github.com/aptos-labs/aptos-core"
```

Comment:
can you keep the tags parameter below the URL like it was originally? not sure why it was changed